### PR TITLE
added button to turn off silence mode

### PR DESCRIPTION
**Pull request by jb08**
@jameswhang code review por favor.

### CODE REVIEW

File: client/www/scripts/application.js [hunk 1]
```diff
@@ -2,12 +2,33 @@ angular.module('SteroidsApplication', [
     'supersonic',
     'firebase'
 ])
+
 .controller('IndexController', function($scope, supersonic) {
     $scope.navbarTitle = "Home";
+
+    $scope.turn_off_silence_mode = function() {
+        supersonic.logger.log("turn_off_silence_mode() called");
```
Comment (jameswhang) [inline client/www/scripts/application.js]: Useless logs

File: client/www/scripts/application.js [hunk 2]
```diff
@@ -2,12 +2,33 @@ angular.module('SteroidsApplication', [
...
+
+        currentTime = new Date();
+        expireTime = currentTime;
```
Comment (jameswhang) [inline client/www/scripts/application.js]: Can probably reduce these down to:
expireTime = new Date();

File: client/www/scripts/application.js [hunk 3]
```diff
@@ -2,12 +2,33 @@ angular.module('SteroidsApplication', [
...
+        expireTime.setMinutes(currentTime.getMinutes() -1);
```
Comment (jameswhang) [inline client/www/scripts/application.js]: and change this to 
expireTime.setMinutes(expireTime.getMinutes() -1);

File: client/www/scripts/application.js [hunk 4]
```diff
@@ -2,12 +2,33 @@ angular.module('SteroidsApplication', [
...
+
+        var piRef = new Firebase('https://bellacoola.firebaseio.com/pi/');
```
Comment (jameswhang) [inline client/www/scripts/application.js:11]: good JS practice is to declare all variables in the beginning of the function when the function is short

File: client/www/index.html [hunk 5]
```diff
@@ -54,10 +53,13 @@
...
         <button class="button button-full button-positive">
-            Weekly Settings 
+            Weekly Settings
         </button>
         </super-navigate>
-
-    </div>
+        
+        <button class="button button-full button-assertive" ng-click="turn_off_silence_mode()">
+            Turn Off Silence Mode
```
Comment (jameswhang) [inline client/www/index.html:61]: there may be a indentation change here that you don't want..?

File: client/www/index.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<p><!DOCTYPE html>
```
Comment (jameswhang) [inline client/www/index.html]: Get rid of this p tag

File: client/www/scripts/application.js [hunk 7]
```diff
@@ -7,28 +7,24 @@ angular.module('SteroidsApplication', [
...
-        supersonic.logger.log("turn_off_silence_mode() called");
-
-        currentTime = new Date();
-        expireTime = currentTime;
-        expireTime.setMinutes(currentTime.getMinutes() -1);
-
+       
         var piRef = new Firebase('https://bellacoola.firebaseio.com/pi/');
 
+        expireTime = new Date()
```
Comment (jameswhang) [inline client/www/scripts/application.js]: Missing semicolon

**Comment by jb08** [comment]
@jameswhang , ok to merge?

**Comment by jameswhang** [comment]
Yup

On Monday, May 9, 2016, Jason B. notifications@github.com wrote:

> @jameswhang https://github.com/jameswhang , ok to merge?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/eecs394-s16/Bellacoola/pull/6#issuecomment-217903049

## 

James(Sung Yoon) Whang
CS | MEAS '2017
Murphy Scholar @ Murphy Institute
Northwestern University

**Pull request merged**